examples/inspec-linux-audit/README.md

```diff
@@ -0,0 +1,32 @@
+# Linux Effortless Audit Example
+
+This example is meant to serve as an example of testing a Habitat + Chef InSpec
+Scaffolding package. It works by sharing `results/` with a Kitchen VM and then
+running `test/scripts/provisioner.sh` which will:
+
+  - Install the Habitat CLI in the Kitchen instance
+  - Configure the `hab` user and group
+  - Install the latest Habitat `.hart` package in `/tmp/results`
+    - Don't forget to run build your package on each change!
+  - Run `inspec` inside the package to converge the Kitchen instance
+
+## Prerequisites
+
+1. [Install Chef Workstation](https://downloads.chef.io/chef-workstation/)
+
+2. [Install Chef Habitat](https://www.habitat.sh/docs/install-habitat/)
+
+3. Run `hab setup` and follow the prompts. The origin you create will be your
```

Comment:
This is a good idea but, why are we not building inside the kitchen? (inside the vagrant box that tk starts)

Reply:
In my opinion, the reason we don't build in kitchen because we don't need to build in kitchen because the docker studio provides enough isolation and we want the habitat experience for effortless to be the same as habitat for applications.

Reply:
Building outside of kitchen is a good idea, as it provides build level isolation away from the test machine. This also follows how our CI works, and how most (all?) users expect Effortless to function.